config/config.walloon.yaml

```diff
@@ -35,14 +33,21 @@ countries:
 - LU
 
 electricity:
+  custom_powerplants: True
   walloon_reassignment: True
+  powerplants_filter: (DateOut >= 2024 and DateOut != DateOut) and not (Country in ['BE'] and Fueltype == 'Nuclear')
+
+existing_capacities:
+  grouping_years_power: [1920, 1950, 1955, 1960, 1965, 1970, 1975, 1980, 1985, 1990, 1995, 2000, 2005, 2010, 2015, 2020, 2025, 2030, 2035, 2040]
 
 # docs in https://pypsa-eur.readthedocs.io/en/latest/configuration.html#clustering
 clustering:
   mode: custom_busmap_BE
   temporal:
     resolution_elec: False
     resolution_sector: 3h
+  exclude_carriers:
+    - nuclear
```

Comment:
I thought we want to remove nuclear? I'd suggest adding here the full list from default, but without `nuclear`